engine/src/main/java/org/entando/entando/aps/system/services/widgettype/cache/WidgetTypeManagerCacheWrapper.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2018-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.aps.system.services.widgettype.cache;
+
+import com.agiletec.aps.system.exception.ApsSystemException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import org.entando.entando.aps.system.services.widgettype.IWidgetTypeDAO;
+import org.entando.entando.aps.system.services.widgettype.WidgetType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.cache.Cache;
+import org.springframework.cache.CacheManager;
+
+/**
+ * @author E.Santoboni
+ */
+public class WidgetTypeManagerCacheWrapper implements IWidgetTypeManagerCacheWrapper {
+
+	private static final Logger _logger = LoggerFactory.getLogger(WidgetTypeManagerCacheWrapper.class);
+
+	private CacheManager _springCacheManager;
+
+	@Override
+	public void loadWidgetTypes(IWidgetTypeDAO widgetTypeDAO) throws ApsSystemException {

```

Comment:
Same question as the other load method on the cache manager. I'm probably missing something but if we're evicting on this call i think the methods should be named something much more explicit so that some dev in the future doesn't make this call in a normal workflow and wipes out the cache over and over again. Something like `public void clearCacheAndReloadWidgetTypes (...)`

Reply:
I try to rename the method and split it into some methods